package.json

```diff
@@ -5,7 +5,7 @@
     "type": "git",
     "url": "https://github.com/StackExchange/Stacks.git"
   },
-  "version": "0.16.0",
+  "version": "0.16.1",
```

Comment:
This needs to be updated in the `package-lock.json` file as well. Make sure you're running `npm version 0.16.1`

Reply:
I am. Not sure why `package-lock.json` isn't getting updated.